topics/galaxy-interface/metadata.yaml

```diff
@@ -11,7 +11,7 @@ subtopics:
     title: "Uploading Data"
     description: "These tutorials will help you get data into Galaxy, before you analyse it."
   - id: interface
-    title: "Galaxy Interface"
+    title: "Galaxy Features"
     description: "These tutorials will cover the UI and important features"
```

Comment:
UI is even less accessible than interface! Maybe "Learn about the most useful Galaxy features" or something like that?

Reply:
yes, that was the point of this pr? removing the "UI" from the homepage.

The description can definitely be better (but no one reads those since they're on the next page)

Reply:
"These tutorials will cover how to navigate Galaxy efficiently" or something?